### PR TITLE
(Re-)combine candidate genes and variants

### PR DESCRIPTION
**Pull request by buske**
Attempt to resolve issue #62.

### CODE REVIEW

**Comment by fschiettecatte** [comment]
+1

File: search-api.md [hunk 1]
```diff
@@ -61,22 +61,24 @@ After receiving a request, the remote server can respond in one of two ways:
...
+        "gene" : {
+          "id" : <gene symbol>|<ensembl gene ID>|<entrez gene ID>
+        },
+        "variant" : {
+          "assembly" : "NCBI36"|"GRCh37.p13"|"GRCh38.p1"|…,
+          "referenceName" : "1"|"2"|…|"X"|"Y",
+          "start" : <number>,
+          "end" : <number>,
+          "referenceBases" : "A"|"ACG"|…,
+          "alternateBases" : "A"|"ACG"|…,
```
Comment (sdumitriu) [inline search-api.md]: Extra comma at the end.

Comment (buske) [inline search-api.md]: Your attention to detail is amazing.

**Comment by Relequestual** [comment]
+1

**Comment by buske** [comment]
That's 3!

**Pull request merged**